drupal/sites/all/features/boinc_standard/boinc_standard.features.menu_links.inc

```diff
@@ -82,11 +82,10 @@ function boinc_standard_menu_default_menu_links() {
   );
   // Translatables
   // Included for use with string extractors like potx.
```

Comment:
I suspect this comment is automatically generated by the Drupal Features module/feature, which adds text wrapped with `t()` to the bottom of Feature's code & files. Do you recommend removing them as I find them?